### ISSUE TITLE
Build 6 - A non numeric character will restart the application

### ISSUE THREAD

**Issue by SeeMoreGain** [TestSheepNZ/TestSheepNZ.github.io#112]

**Comment by SrbaGitHub** [comment]
I agree with you, I get the same result. After submitting "ppp" as a correct answer I get a feedback message: ERROR, NaN not handled and then the page was restarted.

**Comment by yuan-liru** [comment]
The feedback message for number 15 was: Too low and the feedback message for the number 16 was: Too high.

**Comment by yuan-liru** [comment]
For basic calculator part. when i input first number 20, and second number 40, operation is divide. The feedback message for result is 0.
